net2/TagManager.js

```diff
@@ -110,6 +110,22 @@ class TagManager {
         await rclient.delAsync(key);
         this.subscriber.publish("DiscoveryEvent", "Tags:Updated", null, this.tags[uid].o);
         await this.refreshTags();
+
```

Comment:
Better to move these logics into Tag.destroyEnv

Reply:
The FireAPI process does not call Tag.destroyEnv

Reply:
> Better to move these logics into Tag.destroyEnv

The FireAPI process does not call Tag.destroyEnv

Reply:
It can be done in the background